src/constants.ts

```diff
@@ -0,0 +1 @@
+export const localStorageKey = 'token';
```

Comment:
If this key is used for authentication only, please define more specified variable name for the variable, like `authLocalStoragekey`.  I think the `token` value is okay.

Reply:
yes, good idea!